### DISCUSSION TITLE
Gopro labs firmware wont download into gopro?

### DISCUSSION THREAD

**Discussion by jellingwood1821** [gopro/labs#728] [Q&A]
I can not get the firmware downloaded to my gopro hero 11. I have done a reset and reformatted the sd card any help will be appreciated

**Comment by dnewman-gpsw** [comment]
Follow these instructions https://gopro.github.io/labs/install/

**Comment by jellingwood1821** [comment]
Thank you for your response.Sent from my iPhoneThis is what it looks like. But the folder says empty even though I extracted it, I also tried dragging and coping. Any help is greatly appreciated On Jan 21, 2024, at 2:32 PM, David Newman ***@***.***> wrote:﻿
Follow these instructions https://gopro.github.io/labs/install/

**Reply by dnewman-gpsw** [reply]
Reply contains no image.  Likely an issue on your PC or Mac.  The zip contains 200+Mbytes of data, three files within the UPDATE folder.
